app/services/patient_action_or_outcome_service.rb

```diff
@@ -0,0 +1,26 @@
+class PatientActionOrOutcomeService
+  def self.call(consent:, triage:, vaccination_record:)
+    if consent.nil?
+      { action: :get_consent }
+    elsif consent&.consent_refused?
+      { action: :check_refusal }
+    elsif vaccination_record&.administered?
+      { outcome: :vaccinated }
+    elsif !vaccination_record&.administered.nil? &&
+          !vaccination_record&.administered?
```

Comment:
You don't need the additional `!vaccination_record&.administered?` here since we've already accounted for that above, no? Or are you still trying to be explicit?

Reply:
you're correct, i've tweaked it in [b7e5d42](https://github.com/nhsuk/record-childrens-vaccinations/pull/280/commits/b7e5d42fefa2626998e9bfc3aff2788047fcfad1). I still personally prefer being explicit but if we're leaning this way of doing it, better be consistent.